packs/data/journals.db/archetypes.json

```diff
@@ -13,8 +13,9 @@
n</p><hr /><p>You can squeeze out of tight situations surprisingly quickly, gaining an advantage against foes that try to pin you down. You gain the Quick Squeeze skill feat, and if you're a master in Acrobatics, you can Squeeze at full Speed. Whenever you successfully Escape using Acrobatics, the creature you Escaped from is flat-footed against the next attack you make against it before the end of your next turn.</p><h2>@Compendium[pf2e.feats-srd.Dodge Away]{Dodge Away} Feat 6</h2><p><strong>Prerequisites</strong> Acrobat Dedication</p><p><strong>Trigger</strong> You are the target of a melee attack.</p><p><strong>Requirements</strong> You're aware of the attack and aren't flat‑footed.</p><hr /><p>You use your acrobatic prowess to evade an attack, using momentum to keep yourself moving, if you choose. You gain a +1 circumstance bonus to AC against the triggering attack. If the attack misses you, you can Step after the Strike. If you're a master in Acrobatics, you can move 10 feet on this Step instead of 5 feet.</p><h2>@Compendium[pf2e.feats-srd.Graceful Leaper]{Graceful Leaper} Feat 7</h2><p><strong>Prerequisites</strong> Acrobat Dedication, master in Acrobatics</p><hr /><p>Mass and muscle are meaningless when you leap; only grace and balance matter. You can roll an Acrobatics check instead of an Athletics check when making a High Jump or Long Jump.</p><h2>@Compendium[pf2e.feats-srd.Tumbling Strike]{Tumbling Strike} Feat 8</h2><p><strong>Prerequisites</strong> Acrobat Dedication</p><p><strong>Requirements</strong> You are adjacent to an enemy.</p><hr /><p>Attempt an Acrobatics check against the Reflex DC of an enemy adjacent to you.</p><hr /><p><strong>Critical Success</strong> You move through the enemy's space to an unoccupied space on the other side of the enemy from your starting position. This movement doesn't trigger reactions. You can't move farther than your Speed, and you must end your movement adjacent to the enemy whose space you moved through. After moving, you make a melee Strike against the enemy whose space you moved through, and the enemy is flat-footed against that Strike.</p><p><strong>Success</strong> As critical success, but the enemy isn't flat-footed against the Strike.</p><p><strong>Failure</strong> You remain in your original space but can still Strike.</p><p><strong>Critical Failure</strong> No effect.</p><h2>@Compendium[pf2e.feats-srd.Tumbling Opportunist]{Tumbling Opportunist} Feat 10</h2><p><strong>Prerequisites</strong> Acrobat Dedication</p><p><strong>Frequency</strong> once per minute</p><p><strong>Requirements</strong> Your most recent action was to Tumble Through or Tumbling Strike, and you successfully moved through an enemy's space.</p><p>You use a burst of stamina to perform a breathtaking feat of Acrobatics as you speed through a foe's space, leaving your foe lying flat on their back. You attempt to Trip the enemy whose space you moved through. You can use Acrobatics instead of Athletics for this check.</p>",
```

Comment:
It looks like the only thing that is different on these is the new line characters are removed. We can exclude these changes from this and the changes to other existing items in the journal.